src/schema/v2/homeView/sections/NewWorksForYou.ts

```diff
@@ -9,7 +9,7 @@ export const NewWorksForYou: HomeViewSection = {
   type: HomeViewSectionTypeNames.HomeViewSectionArtworks,
   contextModule: ContextModule.newWorksForYouRail,
   component: {
-    title: "New Works for You",
```

Comment:
A/B test description: https://www.figma.com/design/3LYkzfvfTPcYAIgzkaOM1C/Personalization?node-id=120-14313&m=dev

Reply:
FYI - we're deferring the A/B test to avoid conflicting with other running
experiments. We'll merge these other "release and monitor" copy changes in the meantime.